stratum/hal/bin/barefoot/README.md

```diff
@@ -264,39 +223,3 @@ To start a shell session, you can use (requires Docker):
 
 Refer to the [p4runtime-shell](https://github.com/p4lang/p4runtime-shell)
 documentation for more information.
-
-## Building and installing the Debian package
-
-### Prerequests
-
-Before building the Debian package, you need to build SDE and/or the BSP, see this [section](#installing-the-sde) for more information.
-
-To build Debian package, use command below:
-
-```
- bazel build //stratum/hal/bin/barefoot:stratum_bf_deb [--define sde_ver=8.9.2] [--define phal_with_onlp=false]
-```
-
-For more information of `sde_ver` and `phal_with_onlp`, see [Specify dependency for a different SDE version](#specify-dependency-for-a-different-sde-version) and [Disable the ONLP support](#disable-the-onlp-support)
-
-You can find the Debian package here:
-`bazel-bin/stratum/hal/bin/barefoot/stratum_bf_deb.deb`
-
-The Debian package includes some necessary libraries like PI so you don't need to install these libraries into the target device.
-
-To install the package, simply upload the deb file to your device and use `apt` command:
-
-```bash
-[sudo] apt update
```

Comment:
We should keep this.
A completely fresh switch has no cache and will not resolve the dependencies.